docs/guides/0x-animate-your-visualizations.md

```diff
@@ -0,0 +1,391 @@
+
+```js
+filter: animation($date_time, 30, fade(0, 0.5))
+```
+
+<div class="example-map">
+    <iframe
+        id="animation-step-3"
+        src="/developers/carto-vl/examples/maps/guides/animation/step-3.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+#### Final touches
+
+Since this visualization represents the journey of three different birds, we can assign  a unique color to  each one  using their names to define `buckets` inside of a `ramp`.decrease the `width` of the points to `4` and finally, remove the `strokeWidth` by setting it to `0`. 
+
+*Note:*
+For a more in-depth discussion of ramps and other styling properties, check out our [Data Driven Visualizations Guide](/developers/carto-vl/guides/0x-data-driven-visualizations.md). 
+
+```js
+const viz = new carto.Viz(`
+    filter: animation($date_time, 10, fade(0, 0.5))
+    color: ramp(buckets($bird_name, ["Sanne", "Eric", "Nico"]),[deeppink, yellow, turquoise])
+    width: 4
+    strokeWidth: 0
+`);
+```
+
+<div class="example-map">
+    <iframe
+        id="animation-step-4"
+        src="/developers/carto-vl/examples/maps/guides/animation/step-4.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+### Adding controls to your animation
+
+Congratulations! You have created an animated visualization. 
+
+Next, let’s add some animation controls to play and pause the animation as well as a slider to adjust the speed of the animation on-the-fly. 
+#### Add variables
+
+Continuing with our map of bird migration, we will add  two variables in our Viz object: `@duration` and `@animation`. The `@duration` variable defines how to modify the duration value, which is really useful for playing with the animation expression.
+```js
+const viz = new carto.Viz(`
+    @duration: 30
+    @animation: animation($date_time, @duration, fade(0, 0.5))
+    filter: @animation
+`);
+```
+
+The different variables are available in the `viz` object you have just created. If you want to access the animation variable you can do it by typing `viz.variables.animation`. By assigning the animation expression to the `@animation` variable, you are going to be able to call the [public methods](http://animation-guide.developers.carto-staging.com/developers/carto-vl/reference/#expressionsanimation) as follows:
+
+```js
+viz.variables.animation.play();
+```
+
+#### Add control elements
+
+The next step is to add buttons and elements that allow you to control and interact with the animation. 
+
+The   elements we will add are:
+
+* **Progress** slider to display the animation progress.
+* **Play** button to play or resume the animation.
+* **Pause** button to pause the animation.
+* **Duration** slider to control the speed of the animation.
+
+```html
+<aside>
```

Comment:
It would be nice to tell the reader something introducing this snippet. "To show the controls, we're going to create a panel. You have to add this just under the map div...."...